client/client.go

```diff
@@ -36,10 +39,18 @@ type Client interface {
 	Peers(context.Context, ...rpc.Option) ([]info.Peer, error)
 	GetContractInfo(ethcommon.Address, bool) (string, uint8, error)
 	CallContract(context.Context, interfaces.CallMsg, *big.Int) ([]byte, error)
+	GetNetworkID(context.Context, ...rpc.Option) (uint32, error)
+	GetBlockchainID(context.Context, string, ...rpc.Option) (ids.ID, error)
+	IssueTx(ctx context.Context, txBytes []byte) (ids.ID, error)
+	GetAtomicUTXOs(ctx context.Context, addrs []string, sourceChain string, limit uint32, startAddress, startUTXOID string) ([][]byte, api.Index, error)
+	EstimateBaseFee(ctx context.Context) (*big.Int, error)
 }
 
+type EvmClient evm.Client
```

Comment:
We were following the existing embedding pattern in `client.go`.   

`Client` is already used by the existing `info.Client` so aliasing allowed us to prevent the conflict.